calico_cni_k8s_test.go

```diff
@@ -969,7 +969,7 @@ var _ = Describe("CalicoCni", func() {
 
 				checkIPAMReservation := func() {
 					// IPAM reservation should still be in place.
-					ipamIPs, err := calicoClient.IPAM().IPsByHandle(workloadName)
+					ipamIPs, err := calicoClient.IPAM().IPsByHandle(contNs.Path() + "." + containerID)
```

Comment:
`contNs.path()` isn't what we want to use here.

The key information should be some form of `(cni_network_name, containerID)`